owid/catalog/utils.py

```diff
@@ -1,12 +1,39 @@
+import re
 from typing import Optional
+
 from .tables import Table
 
 
 def underscore(name: Optional[str]) -> Optional[str]:
+    """Convert arbitrary string to under_score. This was fine tuned on WDI bank column names.
+    This function might evolve in the future, so make sure to have your use cases in tests
+    or rather underscore your columns yourself.
+    """
     if name is None:
         return None
 
-    name = name.replace(" ", "_").replace("-", "_").lower()
+    name = (
+        name.replace(" ", "_")
+        .replace("-", "_")
+        .replace(",", "_")
+        .replace(".", "_")
+        .lower()
+    )
+
+    # replace parantheses with __
+    name = name.replace("(", "__").replace(")", "__")
+
+    # replace special symbols
+    name = name.replace("%", "pct")
+    name = name.replace("+", "plus")
+    name = name.replace("us$", "usd")
+    name = name.replace("$", "dollar")
+
+    # replace quotes
+    name = name.replace("'", "")
+
+    # shrink triple underscore
+    name = re.sub("__+", "__", name)
 
     if "(" in name:
         raise NameError(f"{name} contains forbidden character `(`")
```

Comment:
This risks letting tons of new edge cases slip through. Instead, the if statement at the end should simply call `validate_underscore` on the normalised name.

Reply:
Oh crap, I forgot it there... Good idea with `validate_underscore`, that's much better.